tools/respecDocWriter.js

```diff
@@ -249,9 +249,20 @@ function makeConsoleMsgHandler(page) {
    */
   return function handleConsoleMessages(whenToHalt) {
     page.on("console", async message => {
+      const text = message.text();
       const type = message.type();
-      const args = await Promise.all(message.args().map(stringifyJSHandle));
-      const text = args.join(" ");
+      if (
+        type === "error" &&
+        !message.args().length && // browser errors have no arguments
```

Comment:
>browser errors have no arguments

IMO that means browser errors are actually distinguishable, no?

Reply:
In some sense, yes... but if someone calls `console.error()` they won't be. That's why I decided to check for the string.